install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/main.yml

```diff
@@ -106,6 +106,7 @@
     chdir: "{{ securedrop_app_code_deb_dir }}"
   environment:
     DH_VIRTUALENV_INSTALL_ROOT: "/opt/venvs"
+    SETUPTOOLS_USE_DISTUTILS: "stdlib"
```

Comment:
This will not fix the issue for us it seems :( The package will be built, but it will fail to install.

```
debconf: unable to initialize frontend: Dialog
debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype
+ set -o pipefail
+ SDBIN=/opt/venvs/securedrop-app-code/bin
+ case \"$1\" in
+ set_paxctld_config
+ paxctld_config=/etc/paxctld.conf
+ '[' -f /etc/paxctld.conf ']'
+ grep -q '^/usr/sbin/apache2' /etc/paxctld.conf
+ systemctl enable paxctld
Synchronizing state of paxctld.service with SysV init with /lib/systemd/systemd-sysv-install...
Executing /lib/systemd/systemd-sysv-install enable paxctld
+ systemctl start paxctld
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/
+ chmod 0700 /var/lib/securedrop/
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/tmp
+ chmod 0700 /var/lib/securedrop/tmp
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/shredder
+ chmod 0700 /var/lib/securedrop/shredder
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/store
+ chmod 0700 /var/lib/securedrop/store
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/keys
+ chmod 0700 /var/lib/securedrop/keys
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop//keys/private-keys-v1.d
+ chmod 0700 /var/lib/securedrop//keys/private-keys-v1.d
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop//keys/openpgp-revocs.d
+ chmod 0700 /var/lib/securedrop//keys/openpgp-revocs.d
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/lib/securedrop/backups
+ chmod 0700 /var/lib/securedrop/backups
+ for dir in '/var/lib/securedrop/{,tmp,shredder,store,keys,/keys/private-keys-v1.d,/keys/openpgp-revocs.d,backups}' /var/www/securedrop
+ mkdir -p /var/www/securedrop
+ chmod 0700 /var/www/securedrop
+ '[' -e /var/lib/securedrop/keys/gpg-agent.conf ']'
+ grep -qE '^allow-loopback-pinentry$' /var/lib/securedrop/keys/gpg-agent.conf
+ '[' '!' -d /var/lib/securedrop/keys/private-keys-v1.d ']'
+ chown -R www-data:www-data /var/lib/securedrop /var/www/securedrop
+ chown -R www-data:www-data /var/www/securedrop
+ chown www-data:www-data /var/www/journalist.wsgi
+ chown www-data:www-data /var/www/source.wsgi
+ a2dissite 000-default
+ a2dissite default-ssl
+ service apache2 stop
+ /opt/venvs/securedrop-app-code/bin/mod_wsgi-express module-config
Traceback (most recent call last):
  File \"/opt/venvs/securedrop-app-code/bin/mod_wsgi-express\", line 33, in <module>
    sys.exit(load_entry_point('mod-wsgi==4.6.7', 'console_scripts', 'mod_wsgi-express')())
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 2855, in load_entry_point
    return ep.load()
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 2446, in load
    return self.resolve()
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/pkg_resources/__init__.py\", line 2452, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/mod_wsgi/server/__init__.py\", line 35, in <module>
    import distutils.sysconfig
  File \"<frozen importlib._bootstrap>\", line 969, in _find_and_load
  File \"<frozen importlib._bootstrap>\", line 958, in _find_and_load_unlocked
  File \"<frozen importlib._bootstrap>\", line 666, in _load_unlocked
  File \"<frozen importlib._bootstrap>\", line 577, in module_from_spec
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/_distutils_hack/__init__.py\", line 82, in create_module
    return importlib.import_module('._distutils', 'setuptools')
  File \"/opt/venvs/securedrop-app-code/lib/python3.5/importlib/__init__.py\", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File \"<frozen importlib._bootstrap>\", line 981, in _gcd_import
  File \"<frozen importlib._bootstrap>\", line 931, in _sanity_check
SystemError: Parent module 'setuptools' not loaded, cannot perform relative import
dpkg: error processing package securedrop-app-code (--install):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:

```